ansible/reset_local_environment.yml

```diff
@@ -12,9 +12,9 @@
       private: no
   roles:
     - { role: openshift_setup, reset_cluster: True }
-    - env_hacks
     - service_catalog_setup
     - ansible_service_broker_setup
+    - env_hacks
```

Comment:
@jwmatthews We thought that moving it prior to the Ansible Service Broker Setup role made sense because etcd uses a persistent volume now with the new changes to the template. When etcd first comes up and env_hacks is at the end then the PV directory will not have enough permissions to initially run. I'm not sure if you see any problems running it at the end but I would assume the etcd pod restarts a couple times if the initial mount fails.